src/App.js

```diff
@@ -1,33 +1,20 @@
 import React from 'react';
-import {BrowserRouter as Router, Switch, Route, Link } from 'react-router-dom';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import './App.css';
+import AdminFrontPage from './components/admin/Admin';
 
 function App() {
   return (
     <Router>
       <div>
-        <nav>
```

Comment:
don't we need this anymore?

Reply:
No necessarily, those were just links on the front page that redirect to `/admin` and `/form`, but those can be accessed by just adding /admin or /form to the url. Those routes added so I and Alicja don't have merge conflicts for the main file of the project, but we've realised the links themselves are redundant.

@LightFelicis has offered to create a prototype for the authentication page, meaning she'll add routes from that page to the admin interface and the form interface, so she'll develop this blank main page we have currently.

Reply:
Ok

Reply:
I will create a ticket fot this part now.